ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -98,28 +174,30 @@ const BaseLocusListLabels = React.memo((
   ) : (
     <div style={containerStyle}>
       {locusListGuids.map((locusListGuid) => {
-        const panelAppConfidence = locusListConfidence && locusListConfidence[locusListGuid]
-        let { description } = locusListsByGuid[locusListGuid] || {}
-        if (panelAppConfidence) {
-          description = (
-            <div>
-              {description}
-              <br />
-              <br />
-              <b>PanelApp gene confidence: &nbsp;</b>
-              {PANEL_APP_CONFIDENCE_DESCRIPTION[panelAppConfidence]}
-            </div>
-          )
+        const locusList = locusListsByGuid[locusListGuid]
+        const panelAppDetails = panelAppDetail[locusListGuid]
+        const { name: label, description: locusListDescription, paLocusList } = locusList
+        const { description, initials, customColor } = panelAppDetails && paLocusList ? getPaProps({
```

Comment:
not strictly necessary, but I think parenthesis would make the control flow here a little cleaner. The `&& ? :` is a little hard to read

Reply:
Ok 👍  I thought the linter would have added that, but it didn't. I'll put it in.